templates/modern-solutions/rop-chain/statement.rst

```diff
@@ -0,0 +1,8 @@
+Open the session task archive and access the ``rop-chain/`` subfolder. You goal
+is to call ``mega_checker()`` after the ``checker()`` call. Both the
+``"ihahaha!"`` and the ``"Uberihahaha!"`` messages need to be printed. For the
+``mega_checker()`` to be properly called you need to initialized both the RDI
+register (1st parameter) and the RSI register (2nd parameter).
+
+Construct the ROP-based payload in ``exploit.py``.
+
```

Comment:
Remove extra blank line.

Reply:
I think it would be OK if a version of this challenge were running remotely and you would use pwntools for that.

Reply:
Sounds ok. I will try to spawn a docker tonight.

Reply:
I won't have time to configure a docker image for the task. We could add this as a nice-to-have for next year.